src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/P11KeyAgreement.java

```diff
@@ -73,11 +73,8 @@ private static class AllowKDF {
 
         private static final boolean VALUE = getValue();
 
-        @SuppressWarnings("removal")
         private static boolean getValue() {
-            return AccessController.doPrivileged(
-                (PrivilegedAction<Boolean>)
-                () -> Boolean.getBoolean("jdk.crypto.KeyAgreement.legacyKDF"));
+            return Boolean.getBoolean("jdk.crypto.KeyAgreement.legacyKDF");
```

Comment:
I'm fine with the code as is, but is there a reason why `Boolean.getBoolean(...)` is not replacing the method call on line 74?  Is there initialization/startup concerns?

Reply:
I removed the `getValue` method and changed VALUE to call `getBoolean`. It's also debatable whether it needs to be lazy initialized in a static class, but that is getting a bit beyond the scope of these changes so I left that as-is.